celestia-node.md

```diff
@@ -22,6 +29,26 @@ git clone https://github.com/celestiaorg/celestia-node.git
 make install
 ```
 
+## Understanding config.toml
+After initialization, for any type of node, you will find a `config.toml`. Let's break down some of the most used sections
+### [Core]
+This section is needed for the Celestia Full Node. By default, `Remote = false`. Still for devnet, we are going to use the remote core option and this can also be set
+by the command line flag `--core.remote` 
+
+### [P2P]
+#### Bootstrap
+By default, the `Bootstrapper = false` and the `BootstrapPeers` is empty. You do need to activate this section and add bootstrap peers in order to bootstrap your node
+
+#### Mutual Peers
+The purpose of this config is to set up a bidirectional communication. This is usually the case for Celestia Full Nodes. In addition, you need to change the field 
+`PeerExchange` from false to true
+
+### [Services]
+#### TrustedHash and TrustedPeer
+TrustedHash is needed to properly initialize Celestia Full Node with the active `Remote` Core Client. Celestia Light Node needs to be initialized with the trusted hash, too.
+TrustedPeer is the most crucial field to be filled for correct Celestia Light Node execution. Any Celestia Full Node can be a trusted peer for the Light one. However, the Light node
```

Comment:
Reading this I'm realizing that this is trusted peer might not be the best name as the light client does not really need to trust any peer.

Reply:
We can think about this in the celestia-node issue part. Are you ok if we transfer the further discussion into the other space?

Reply:
@liamsi, as per devnet we defined that we rely only on a configured set of peers for HeaderSync which were called trustedPeers. We trust them not just as a reliable source of header availability but also as the latest chain head providers. Nothing changed in this since devnet and we are still relying on a strict set of trustedPeers. Going further, we will change this to be able to request headers from any peer, but the trusted peers would still be in use as the source of trusted latest chain head for automatic subjective initialization, in case our subjective latest head is expired.

Reply:
So OK, they are called trusted because you trust them for the subjective head. IMO, it would still be good if the light nodes at least would also do what tendermint does: verify header hashes with secondary nodes.